EDMarketConnector.py

```diff
@@ -330,6 +332,18 @@ def __init__(self, master: tk.Tk):  # noqa: C901, CCR001 # TODO - can possibly f
 
         self.prefsdialog = None
 
+        self.only_tray_close = 0
```

Comment:
Please clarify how this is used in a comment (I saw you're planning some commits for that), as its unclear on first look over.

Reply:
Done. Probably a bit too much but still.